pkg/kubectl/resource_printer.go

```diff
@@ -2447,6 +2533,20 @@ func (j *JSONPathPrinter) PrintObj(obj runtime.Object, w io.Writer) error {
 		}
 	}
 
+	if unknown, ok := obj.(*runtime.Unknown); ok {
```

Comment:
> twitch

Found inside a list somewhere I think.  Given the way this code works, the change here is actually correct.  Well, more correct than it was before.  The whole thing needs some rethinking.